gcf/lib/path.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package lib
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"path/filepath"
+	"strings"
+)
+
+// DataFiles represent a logical unit of files for a single dataset.
+type DataFiles struct {
+	TMCFPath string
+	CSVPaths []string
+}
+
+// ImportGroupFiles represent the collection of paths for a single import group.
+type ImportGroupFiles struct {
+	DataSource2DatasetNames map[string][]string
```

Comment:
nit: no need to have "data" in all the variable names. Can just be "Source2Dataset", "Dataset2Files". It makes the code below easier to read.

Reply:
done.